### DISCUSSION TITLE
Business question for a technical audience

### DISCUSSION THREAD

**Discussion by baddoggames** [rustdesk/rustdesk-server-pro#148] [Q&A]
Our company has completed a technical review of RustDesk we like what we see. I have attempted to contact RustDesk to learn more, understand how their business model works and see if they are a fit for our company. I have sent a couple of emails to their sales@rustdesk.com address. First email was a reply with the URL of the license pricing, the second was a like to Github. Has anyone in this forum had a business-relevant conversation with anyone from RustDesk? Is that even an option?

**Comment by rustdesk** [comment]
Can we discuss non confidential business questions here to make high efficiency and productivity? Since we only have a tiny dev team, no sales members.
